.github/workflows/testing.yml

```diff
@@ -77,8 +77,15 @@ jobs:
 
   bazel:
     runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        include:
+          - bazel: 6.0.0
```

Comment:
You should probably put in a comment that one entry needs to have `bzlmod: false` regardless of whether or not the bazel versions are the same.  This looks like 6.0.0 doesn't support bzlmod, so it could easily be dropped if support for 6.0.0 is dropped.